app/src/main/res/values/array.xml

```diff
@@ -41,7 +41,14 @@
         <item>0</item>
         <item>1</item>
     </array>
-
+    
+    <!-- MainAPI enum: AutoDownloadMode -->
+    <array name="auto_download_plugin">
+        <item>Disable</item>
+        <item>Filter by Preferred Language</item>
+        <item>All</item>
+        <item>NSFW Only</item>
+    </array>
```

Comment:
This should be res/strings as only strings are translated, not array.

you can use 
@string/nsfw
@string/all 
@subtitles_filter_lang
and add
@string/disable = disable

Reply:
Done changes